### DISCUSSION TITLE
Combobox or Autocomplete Data Binding

### DISCUSSION THREAD

**Discussion by lormandrw** [microsoft/fluentui-blazor#1499] [Q&A]
I'm new to all this, so be patient with me.

I have 2 model lists created from API calls. 
Model1 stores the properties of an object.  
Model2 stores a list of options with (int ID) and (string Value).

Model1 will store a selected (int ID) value from Mode2.
My goal is to display a Combobox or Autocomplete that will display the options (string Value)s from Model2 and also be bound to Model 1 (int ID) value.  So select the selected Mode2 (int ID) would be bound to Model 1 (int ID) 

This seems like a basic requirement of a model based control, so I'm not sure what I am missing here. I'm still learning all the lingo, so If possible a small working code example would be nice.  Any help would be greatly apricated.

**Comment by dvoituron** [comment]
Hello and welcome to development in Blazor.

I think it would be best to start with some examples.
https://learn.microsoft.com/en-us/training/paths/build-web-apps-with-blazor/

And then you have our presentation on FluentUI Blazor.
https://youtu.be/a0xOqNemRoY

In the end, you will be able to create a `List<int>` in your code and bind these items to the **FluentSelect** or **FLuentAutocomplete** component, depending on whether you need to display the full list or search for some items based on a criterion.

Is it more clear for you?
